Sources/SpeziViews/SpeziViews.docc/SpeziViews.md

```diff
@@ -81,11 +81,19 @@ Default layouts and utilities to automatically adapt your view layouts to dynami
 
 - ``ImageReference``
 
+### Modifiers
+
+- ``SwiftUICore/View/skeletonLoading(replicationCount:repeatInterval:spacing:)``
```

Comment:
I would probably put it in the same group as the `Animations and Visual Effects`,